api/prisma/schema.prisma

```diff
@@ -996,6 +1007,7 @@ enum YesNoEnum {
 enum RuleEnum {
   nameAndDOB
   email
+  emailAndNameAndDOB
```

Comment:
I know we chatted about this but adding it here:
since can act as a catch all, maybe renaming this enum?